src/main/java/org/folio/bulkops/domain/entity/BulkOperation.java

```diff
@@ -23,7 +23,7 @@
 import lombok.NoArgsConstructor;
 
 @Data
-@Builder
+@Builder(toBuilder = true)
```

Comment:
We can avoid creation of toBuilder() method (and as a result, creation of extra-entity of BulkOperation for using builder)  just changing types of counters from Integer to int. As a result we don't need to create extra entity new BulkOperation() and continue using @Builder.

Reply:
Good catch, thank you!